server/keyshare/keyshareserver/server_unversioned_test.go

```diff
@@ -0,0 +1,461 @@
+package keyshareserver
+
+import (
+	"crypto/ecdsa"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/privacybydesign/gabi/signed"
+	irma "github.com/privacybydesign/irmago"
+	"github.com/privacybydesign/irmago/internal/test"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	irma.Logger.SetLevel(logrus.FatalLevel)
+}
+
+func TestUnvServerInvalidMessage(t *testing.T) {
```

Comment:
I never saw `Unv` as abbreviation in code before. Is there a big obligation to just write it in full?

Reply:
No obligation, think you're right, making the code easier to understand is always a good thing.